### ISSUE TITLE
[Feature] Contact us section

### ISSUE THREAD

**Issue by kharshita590** [ShwetKhatri2001/AR-Webstore#57]
### Description

I want to add a contact us section .

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines]
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

**Comment by ShwetKhatri2001** [comment]
Sure, I'd suggest you to create a separate "Contact us" page and add a button/link on the home page redirecting to that page.
You're assigned.

**Comment by ShwetKhatri2001** [comment]
I have decided to go with React App now, so kindly pull the latest master branch and make your changes on that accordingly.

**Comment by ShwetKhatri2001** [comment]
Any updates @kharshita590 ? Please try to complete your changes in next 3 days. 
After that, it will be open to other contributors.

**Comment by akanksha984** [comment]
@ShwetKhatri2001 Please assign this issue to me , I am ready to work on this

**Comment by ShwetKhatri2001** [comment]
@akanksha984  Assigning you this issue since you're done with your previous issues.
Go through the previous comments to know better about the outcomes. Ask queries if you have any.

**Comment by ShwetKhatri2001** [comment]
Hi @akanksha984 , Just want to know if you're open to contributing to the two issues I've assigned.
You were interested to work on those some time back, that's why I assigned you those.

**Comment by akanksha984** [comment]
Yes I want to work on this @ShwetKhatri2001

**Comment by akanksha984** [comment]
I can design the frontend part

**Comment by ShwetKhatri2001** [comment]
Any updates or issues @akanksha984 ?

**Comment by ShwetKhatri2001** [comment]
This issue is open to make enhancements on the work done.

**Comment by AnupK1234** [comment]
Hey!
I would like to work on Contact Us section. 
Kindly assign me this issue.
Thank You

**Comment by ShwetKhatri2001** [comment]
Okay, Go ahead. Are you taking part in any open-source program ?

**Comment by AnupK1234** [comment]
> Okay, Go ahead. Are you taking part in any open-source program ?

Yes, I am taking part in open-source programs such as KWOC, SWOC, Code Peak..

**Comment by AnupK1234** [comment]
In Contact Us, should I create a contact form ?
Or what more features do you wish to implement in contact us section ?

**Comment by ShwetKhatri2001** [comment]
Yes, you can create a contact form taking full name, email of the user and message and the data should be sent to my email ( mentioned on the current contact page ). 
Feature is to implement a free way ( for limited usage ) to send the form data to my mail id .

**Comment by Akshanshkaushal** [comment]
hey @ShwetKhatri2001  , as we know this a AR store so its contact us must be a attractive one with a integrated 3-D model . So you can assign me this task , I can make a interactive contact us section with 3 d visuals.

**Comment by ShwetKhatri2001** [comment]
@Akshanshkaushal. Good to know that you've got some great ideas. But both the issues you have raised are currently assigned to some people. I'll reach out to you in some days when they will be open again for contribution.

**Comment by ShwetKhatri2001** [comment]
Let's give other contributors a chance to work on this issue. 

@Akshanshkaushal if you want to work on this, please complete the assigned issue first.
